src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -92,6 +92,7 @@ public static class BuiltinNodeCategories
         public const string REVIT_PARAMETERS = "Revit.Parameters";
         public const string REVIT_BAKE = "Revit.Bake";
         public const string REVIT_API = "Revit.API";
+        public const string REVIT_ELEMENTS_DIVIDEDPATH_ACTION = "Revit.Elements.DividedPath.Actions";
```

Comment:
Hi @aosyatnik, I'm so sorry for the delay, it's been a very tight week for me. As mentioned above I'd want those unused stuff out of `NodeCategories.cs`. Whatever that is being used in Revit, should be moved into a new `NodeCategories.cs` file in `DynamoDS/DynamoRevit` repository. See comment I left in [your Revit PR](https://github.com/DynamoDS/DynamoRevit/pull/194).